packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -41,19 +41,39 @@
   @include mdc-text-field-helper-text-color($mdc-text-field-helper-text-color);
   @include mdc-text-field-fullwidth-bottom-line-color($mdc-text-field-fullwidth-bottom-line-color);
   @include mdc-text-field-icon-color($mdc-text-field-icon-color);
+  @include mdc-ripple-surface;
+  // Text Field Box intentionally omits press ripple, so each state needs to be specified individually
+  @include mdc-states-base-color($mdc-text-field-ink-color);
+  @include mdc-states-hover-opacity(mdc-states-opacity($mdc-text-field-ink-color, hover));
+  @include mdc-states-focus-opacity(
+    mdc-states-opacity($mdc-text-field-ink-color, focus),
+    $has-nested-focusable-element: true
+  );
+  @include mdc-ripple-radius-bounded;
+  @include mdc-text-field-corner-radius($mdc-text-field-border-radius);
+  @include mdc-text-field-fill-color($mdc-text-field-background);
+  @include mdc-floating-label-float-position($mdc-text-field-label-position-y);
+  @include mdc-floating-label-shake-animation(text-field);
+  @include mdc-floating-label-width($mdc-text-field-icon-padding);
```

Comment:
Note: remember to change this after updating the width mixin in `mdc-floating-label`

Reply:
Complete!